tests/distributed_grids/2d_coarse_grid_01.debug.output

```diff
@@ -64,11 +64,11 @@ DEAL:2d::Checksum: 3932161
            -9.09745693e-01  -8.64258409e-01   0.00000000e+00
            -2.09025431e+00   1.98574162e+00   0.00000000e+00
            -9.09745693e-01   8.64258409e-01   0.00000000e+00
-            2.09025431e+00  -1.98574162e+00   0.00000000e+00
-            2.09025431e+00   1.98574162e+00   0.00000000e+00
-            9.09745693e-01  -8.64258409e-01   0.00000000e+00
-            9.09745693e-01   8.64258409e-01   0.00000000e+00
            -8.34745646e-01  -8.34745646e-01   0.00000000e+00
+            8.34745646e-01  -8.34745646e-01   0.00000000e+00
```

Comment:
These tests just output the first 20 lines (see tests/distributed/coarse_grid_common.h, line 54), and so there are different points/cells that get into the file.